packages/fxa-content-server/app/scripts/templates/settings/avatar_change.mustache

```diff
@@ -10,14 +10,23 @@
 
     <div class="error"></div>
 
-    <nav id="avatar-options">
-      <input type="file" id="imageLoader" name="imageLoader" accept="image/png, image/jpeg"/>
-      <a href="#" id="file">{{#t}}Upload{{/t}}</a>
-      <a href="/settings/avatar/camera" id="camera">{{#t}}Camera{{/t}}</a>
+    <input type="file" id="imageLoader" name="imageLoader" accept="image/png, image/jpeg"/>
+    <div class="button-row save-options">
+      
+      <button class="save-option" id="file">
+          <div class="graphic graphic-upload-option"></div>
+          <p class="name">{{#t}}Upload{{/t}}</p>
+      </button>
+      <button class="save-option" id="camera">
+          <a href="/settings/avatar/camera">
```

Comment:
Wrapping the anchor here in the button makes the link inaccessible using keyboard. Can you unwrap just this link?

Reply:
I will add the changes this week. I am interested to finish this task :) 

Thank you for your feedback, I appreciate it.